components/constants/webui_url_constants.cc

```diff
@@ -74,3 +74,7 @@ const char kSpeedreaderPanelHost[] = "brave-speedreader.top-chrome";
 const char kShortcutsURL[] = "chrome://settings/system/shortcuts";
 const char kChatUIURL[] = "chrome-untrusted://chat/";
 const char kChatUIHost[] = "chat";
+const char kBravePlayerHost[] = "player";
+const char kBravePlayerURL[] = "chrome://player/";
+const char kBravePlayerEmbedHost[] = "player-embed";
+const char kBravePlayerEmbedURL[] = "chrome-untrusted://player-embed/";
```

Comment:
please move these constants and a feature declaration into `//brave/components/player/common` so everything can be accessed from `//third_party/blink`.

We might use it here https://github.com/brave/brave-core/pull/20792/files#diff-b667b30df32c43a1f70284f27ceda45464c02e3b3aee851b5441c2e7a74e0ae2R22

Reply:
Done. you can find them in `//brave/components/brave_player/common/url_constats.h`